### DISCUSSION TITLE
merged data analysis

### DISCUSSION THREAD

**Discussion by GreyRockIQ** [satijalab/seurat#5506] [Q&A]
Hello
Do I have to use integrated data vignette for two replicates that are merged?
Thanks
GreyRock

**Comment by jjo12** [comment]
Hi,

Not really - depending on what you'd like to do. If you want to just display tSNE/UMAP plots, then `split.by` parameter on DimPlot command would be enough.

[https://satijalab.org/seurat/reference/dimplot]

J
